### ISSUE TITLE
Refresh the role with --force

### ISSUE THREAD

**Issue by nrao-nmdp** [jmhale/okta-awscli#122]
**Is your feature request related to a problem? Please describe.**
I have few different roles with in the same account. So when I authenticate with one role, it automatically adds it to the config file. So, if I re-authenticate with --force it refreshes STS but authenticate with same role. I need to manually go and delete the role in config file.
```
$ okta-awscli --okta-profile test --profile test
Enter password: 
Multi-factor Authentication required.
Pick a factor:
[ 0 ] Okta Verify App: SmartPhone_Android: xxxxxxxx
[ 1 ] token:software:totp( OKTA ) : xxxxx@xxxxx.org
Selection: 0
1: arn:aws:iam::xxxxxxxxx:role/xxxxxx-admins
2: arn:aws:iam::xxxxxxxxx:role/xxxxxx-developer
Please select the AWS role: 1
$ okta-awscli --okta-profile test --profile test --force
Enter password: 
Multi-factor Authentication required.
Pick a factor:
[ 0 ] Okta Verify App: SmartPhone_Android: xxxxxxxx
[ 1 ] token:software:totp( OKTA ) : xxxxx@xxxxx.org
Selection: 0
```
**Describe the solution you'd like**
I think solution can be emptying role in config when --force is triggered.

Any help is appreciated, thanks.

**Comment by jmhale** [comment]
Fixed in `v0.4.2`

**Comment by kevinrobbins** [comment]
This problem has re-appeared in v0.5.0.

```
➜ okta-awscli --force
Enter password: 
Registered MFA factors:
1: Okta Verify - Push
2: Okta Verify
Please select the MFA factor: 1
Waiting for push verification...
export AWS_ACCESS_KEY_ID=REDACTED
export AWS_SECRET_ACCESS_KEY=REDACTED
export AWS_SESSION_TOKEN=REDACTED
```
